src/Shell/Task/ModelTask.php

```diff
@@ -732,11 +732,11 @@ public function fieldValidation($schema, $fieldName, array $metaData, $primaryKe
         }
 
         if (in_array($fieldName, (array)$primaryKey)) {
-            $rules['allowEmpty'] = ['create'];
+            $rules['allowEmpty'] = ["'create'"];
         } elseif ($metaData['null'] === true) {
             $rules['allowEmpty'] = [];
         } else {
-            $rules['requirePresence'] = ['create'];
+            $rules['requirePresence'] = ["'create'"];
```

Comment:
Could this potentially result in working templates no longer working as they require quotes now?

Reply:
Yes it potentially can if someone has overridden this method.

So we can document this change and bump bake version accordingly.

Reply:
We'd have to go to 1.6 I think.

Reply:
Yup, fine by me.